File: despatch.WEB/app/src/swagger/models/user-images-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { ImageRequest } from './image-request';
import { UserRequest } from './user-request';
export interface UserImagesRequest extends BaseEntityRequest{
  image?: ImageRequest;
  imageId?: string;
  user?: UserRequest;
  userId?: string;
}

File: despatch.Domain/Models/Default/Request/UserImagesRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace despatch.Domain.Models.Default.Request
{
    public class UserImagesRequest : BaseEntityRequest
    {
        public string UserId { get; set; }
        public string ImageId { get; set; }

        public virtual UserRequest User { get; set; }
        public virtual ImageRequest Image { get; set; }
    }
}
